packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -95,6 +95,7 @@ function InitialsAvatarFallback(props: AvatarProps) {
 
   return (
     <Flex
+      elementDescriptor={descriptors.avatar}
```

Comment:
Do we need this? 

It looks like the parent Flex already has an `avatar` descriptor

Reply:
@nikosdouvlis yes we need this. There is a different `<Flex />`  element in each case.

Reply:
I pushed a change, @raptisj please review the last commit.